addons/edi/models/edi_issues.py

```diff
@@ -34,54 +31,58 @@ class ProjectTask(models.Model):
                                       index=True, ondelete='cascade')
 
 
-class EdiIssue(models.AbstractModel):
-    """EDI Issue-Tracked Object
+class MainIssue(models.AbstractModel):
```

Comment:
Perhaps just call this class `Issue` `BaseIssue`, or `UdesIssue`?  I'm not sure `Main` tell us much about what the class is for (the same would apply to `_name` and `_description`, if you agree).

Reply:
I think is not a change which improves or make things clearer. Maybe because of my english but at this context to me Main and Base are similar in meaning. 
It seems more e personal preference to me , but if you insist happy to make the change.

Reply:
To me, `Base` implies that this is a class that we intend to subclass, but `Main` doesn't have that meaning.

Reply:
I see that you insist, so i will make you happy.